docs/v2/features-overzicht.md

```diff
@@ -163,6 +163,168 @@ In het volgende feature bestand zijn de bovenstaande regels geïllustreerd aan d
+
+Bijvoorbeeld
+```
+  "gezag": [
+    {
+      "type": "TweehoofdigOuderlijkGezag",
+      "ouders": [
+        {
+          "burgerservicenummer": "999999357"
+        },
+        {
+          "burgerservicenummer": "999999345"
+        }
+      ],
+      "minderjarige": {
+          "burgerservicenummer": "999999412"
+      }
+    }
+  ]
+```
+
+### EenhoofdigOuderlijkGezag
+
+Dit type gezag betekent dat één juridische ouder het gezag heeft over de minderjarige. Dit kan voorkomen wanneer er maar één ouder is, maar ook wanneer alleen één van beide ouders het gezag heeft. Het gezag bevat bij EenhoofdigOuderlijkGezag de burgerservicenummers van de minderjarige en van de ouder die het gezag heeft.
+
+Bijvoorbeeld
+```
+  "gezag": [
+    {
+      "type": "EenhoofdigOuderlijkGezag",
+      "ouder": {
+        "burgerservicenummer": "999970124"
+      },
+      "minderjarige": {
+        "burgerservicenummer": "999970185"
+      }
+    }
+  ]
+```
+
+### GezamenlijkGezag
+
+Dit type gezag betekent dat de juridische ouder en iemand anders - geen ouder - het gezag hebben over de minderjarige. 
+
+Dit kan voorkomen wanneer de ouder en diens (ex) partner het gezag hebben. Het gezag bevat in dit geval de burgerservicenummers van de minderjarige, van de ouder en van de (ex)partner die het gezag hebben. De niet-ouder (partner) met gezag wordt geleverd als "derde".
+
+Dit type gezag kan ook voorkomen wanneer er een gerechtelijke uitspraak over is gedaan, waarbij het gezag aan de ouder en iemand anders is toegewezen. In dat geval weten we niet wie die andere persoon of instelling is die het gezag heeft. Het gezag bevat in dit geval de burgerservicenummers van de minderjarige en van de ouder die het gezag heeft.
+
+Bijvoorbeeld bij gezamenlijk gezag van de ouder en diens partner
+```
+  "gezag": [
+    {
+      "type": "GezamenlijkGezag",
+      "ouder": {
+        "burgerservicenummer": "999970124"
+      },
+      "derde": {
+        "burgerservicenummer": "999970136"
+      },
+      "minderjarige": {
+        "burgerservicenummer": "999970161"
+      }
+    }
+  ]
+```
+
+### Voogdij
+
+Dit type gezag betekent dat een niet-ouder het gezag heeft over de minderjarige. Dit kan voorkomen wanneer er een gerechtelijke uitspraak is gedaan over het gezag. In dat geval weten we niet wie het gezag heeft. Dat kan een voogdijinstelling zijn of een of meerdere personen.
+
+Voogdij kan ook voorkomen wanneer de ouder is overleden of onbevoegd is tot gezag. De (ex)partner van de ouder krijgt dan (onder bepaalde condities) van rechtswege het gezag. De niet-ouder (partner) met gezag wordt dan geleverd als "derde".
+
+Bijvoorbeeld bij voogdij na gerechtelijke uitspraak
+```
+  "gezag": [
+    {
+      "type": "Voogdij",
+      "derden": [],
+      "minderjarige": {
+        "burgerservicenummer": "999999515"
```

Comment:
er is toch geen BSN?

Reply:
wel van de minderjarige, niet van de voogd